docs/cloud/connections.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 3
+---
+
+# Peering Connections
+
+*Peering connections* are used  to connect a Dragonfly Cloud private network with a VPC in your cloud account to establish communication between the two networks over private ip space.
+
+Private IP space communication is more secure, performant and reduces data transfer costs incurred by cloud providers.
+
+Once you created a *private network* as described in [Networks](./networks) click the network three dots menu and click + Connect
+
+Continue to [AWS](#aws) or [GCP](#gcp) based on your cloud provider.
+
+
+## AWS
+Specify the *region*, *CIDR*, *account ID* (also called owner ID in AWS) and *VPC ID* of your AWS VPC from where you want to connect and click *Create*.
+
+The connection will be created in an inactive state.
+
+Accept the peering connection in your AWS account console 
```

Comment:
Missing full stop?

Reply:
This paragraph is a bit confusing, is this mean to be "To accept the peering connection in your AWS account console:" or something?